core/jobs/job_options.py

```diff
@@ -63,6 +63,12 @@ def __init__(
             region=feconf.GOOGLE_APP_ENGINE_REGION,
             temp_location=feconf.DATAFLOW_TEMP_LOCATION,
             staging_location=feconf.DATAFLOW_STAGING_LOCATION,
+            # The 'use_runner_v2' is used since some of our jobs require
+            # the v2 of the runner.
+            # The 'enable_recommendations' is used since we want to receive
+            # recommendations for our jobs.
```

Comment:
What sort of recommendations? Maybe explain to the user how to view those.

Reply:
Done.